install-daemon.sh

```diff
@@ -113,6 +113,12 @@ function detect_distro {
 }
 
 function check_os_comp {
+  MACHINE_TYPE=$(uname -m)
+  if [ "${MACHINE_TYPE}" != "x86_64" ]; then # check the architecture
+      print_error "Detected unsupported architecture $MACHINE_TYPE"
+	  print_error "Use 64 bit architecture(x86_64)"
```

Comment:
Is this indented differently than the line above or is my phone just showing it weirdly?

Reply:
This may be my text's editor's fault. In my code it doesn't show up like this.

Reply:
Fixed in github.